content/en/docs/instrumentation/java/automatic/spring-boot.md

```diff
@@ -23,6 +23,81 @@ Spring Boot starter, see
 
 ## Configuration
 
+### Dependency management
+
+A Bill of Material ([BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms)) allows you to fix the versions of dependencies, including transitive ones.
```

Comment:
(to make it sound more important as opposed to an optional thing. even if it is optional, we want to strongly encourage it)
```suggestion
A Bill of Material ([BOM](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#bill-of-materials-bom-poms)) ensures that versions of dependencies (including transitive ones) are aligned.
```